ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/EVPN_MULTICAST_TESTS.yml

```diff
@@ -128,6 +133,10 @@ tenants:
       # Testing multiple pools and ranges to represent 232.0.16.0/20
       underlay_l2_multicast_group_ipv4_pool: 232.0.16.0/31, 232.0.16.2-232.0.19.255, 232.0.20.0/22, 232.0.24.0-232.0.31.255
       underlay_l2_multicast_group_ipv4_pool_offset: 1
+    vxlan_flood_multicast:
+      enabled: true
+      underlay_l2_multicast_group_ipv4_pool: 232.1.0.0-232.1.0.40
+      underlay_l2_multicast_group_ipv4_pool_offset: 1
```

Comment:
Maybe we should make the default value of the offset 1, so we align the group address with the vlan number by default.

Reply:
We have default set as `0` for `evpn_l2_multicast.underlay_l2_multicast_group_ipv4_pool_offset`
https://github.com/aristanetworks/avd/blob/devel/python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml#L189

It might get a bot confusing for the user.

Reply:
we can move default value to 1 for both in 6.0 lets add a TODO